Makefile

```diff
@@ -24,6 +24,9 @@ asmhentai:
 
 bulk:
 	tomoe --bulk doujin.json
+
+pdf:
```

Comment:
I see a new target testing here, but no new workflows was added, If you want to add testing for pdf could you add the new `.github/workflows` ci too?

Reply:
workflow added